ComStats/comstats.py

```diff
@@ -71,3 +71,13 @@ def percentage_t_test(v, weights=None, opts={'distribution': 't'}, one_sided=Fal
         p_values = stats.t.sf(np.abs(score), dof)
     p_values *= (1 if one_sided else 2)
     return p_values, score
+
+def one_sample_z_test(v, population_proportion, tails=2):
+    return one_sample_z_test_simple(v.sum(axis=1), v.shape[1], population_proportion, tails)
+
+def one_sample_z_test_simple(count, sample_size, population_proportion, tails=2):
+    ratios = count/sample_size
```

Comment:
I should probably add a pep8 linter checker but if you just add a space here it's good to go